app/scripts/modules/core/src/pipeline/config/stages/evaluateVariables/EvaluateVariablesExecutionDetails.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import {
+  IExecutionDetailsSectionProps,
+  ExecutionDetailsSection,
+  StageExecutionLogs,
+  StageFailureMessage,
+} from 'core/pipeline';
+
+export interface IEvaluatedVariables {
+  key: string;
+  value: string;
+}
+
+export function EvaluateVariablesExecutionDetails(props: IExecutionDetailsSectionProps) {
```

Comment:
```suggestion
export const EvaluateVariablesExecutionDetails = (props: IExecutionDetailsSectionProps) => {
```

Reply:
I can apply this convention to the other stages that I'm cleaning up too

Reply:
Hmmm... this breaks the `namespace` thing.
```
[ts] Cannot redeclare block-scoped variable 'EvaluateVariablesExecutionDetails'.
```